.gitignore

```diff
@@ -66,3 +66,5 @@ src/mixxx.rc.include
 src/mixxx.res
 src/test/**/*.actual
 res/qrc_mixxx.cc
+/out/build/x64-Debug
+/.vs
```

Comment:
Please remove that commit.
Personal ignores belong into your personal gitignore, not in the Mixxx repo.
(we had this discussion too often already, at the end it was always 'No.' ; )